README.md

```diff
@@ -73,7 +73,7 @@ Here a configuration example:
 
 ```yaml
 m6_web_amqp:
-    sandbox: false
+    sandbox: false #optional - default false
```

Comment:
I don't understand - why do you need static configuration for tests when usually there are unique test cases. What's the point of duplicating configuration?